fix/fixer_vsphere_network_storage.go

```diff
@@ -8,6 +8,10 @@ import (
 // changes the disk_size, disk_thin_provisioned, and disk_eagerly_scrub into a storage adapter
 type FixerVSphereNetworkDisk struct{}
 
+func (FixerVSphereNetworkDisk) DeprecatedOptions() []string {
+	return []string{"network_card", "network", "networkCard"}
```

Comment:
missed the disk_size, disk_thin_provisioned, and disk_eagerly_scrub

Reply:
Good catch @jhawk28